### ISSUE TITLE
Dependency version mismatch

### ISSUE THREAD

**Issue by ferki** [whosgonna/Zabbix-Tiny.pm#16]
When I try to install Zabbix::Tiny-1.06, I get the following error:

```
$ cpanm Zabbix::Tiny
--> Working on Zabbix::Tiny
Fetching http://www.cpan.org/authors/id/W/WH/WHOSGONNA/Zabbix-Tiny-1.06.tar.gz ... OK
Configuring Zabbix-Tiny-1.06 ... OK
==> Found dependencies: Test::LWP::UserAgent
Found Test::LWP::UserAgent 0.030 which doesn't satisfy 0.30.
! Installing the dependencies failed: Installed version (0.030) of Test::LWP::UserAgent is not in range '0.30'
! Bailing out the installation for Zabbix-Tiny-1.06.
```

**Comment by whosgonna** [comment]
Yes. Looks like i missed a decimal place. :(

Won't take too long to fix, though. I should have an update later today.

**Comment by ferki** [comment]
Great, thank you!

I wanted to send a PR to fix that, but then I noticed that all the meta files are generated (by Minilla, right?), so I wasn't sure how it's best to do and send the fix. Can you point me to the contribution guide I might have missed, please?

**Comment by whosgonna** [comment]
I don't have a contribution guide.  Pull requests are certainly welcome. 

Meta files are mostly generated by Minilla, but it looks like maybe it's hit or miss.  I believe that I added the requirement for Test::LWP::UserAgent.  In fact, i think i added all of the requirements to the cpan file, save for `requires 'Test::More', '0.98';`.  It could be that I'm not using Minilla correctly, or just a limitation.  Either way, this mistake is a manual one that I made.

As far as this issue goes, I'm removing the version requirement for Test::LWP::UserAgent.  I'm not 100% that the syntax is correct. I  think it's just `requires 'Test::LWP::UserAgent';` vs requires 'Test::LWP::UserAgent', '>=0.030';`, so i'm testing on another machine before I commit.  Since you're responding right now, I'll try to add the new tarball here, as it would take a few hours to replicate to CPAN.

**Comment by whosgonna** [comment]
Can you try installing the tarball from this commit?  It worked for me, but I'd like confirmation before I push it to CPAN.

`cpanm Zabbix-Tiny--1.07.tar.gz`

**Comment by ferki** [comment]
I cleaned up stuff, pulled your changes, then ran `minil dist`. Running `cpanm Zabbix-Tiny-1.07.tar.gz` seems to be pulling in the dependency correctly, and then installs correctly afterwards. My scripts using it are still working too :)

It seems to be fixed, thank you!

**Comment by whosgonna** [comment]
Fixed in de2a87ab0f8b801a93a40b51e6e8f1fc401bd99c . New release pushed to CPAN.
